### PR TITLE
add: `systemcoding-conky`

### PR DESCRIPTION
**Pull request by systemcoding**

### CODE REVIEW

File: packages/systemcoding-conky/systemcoding-conky.pacscript [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+prepare() {
+          true
+}
+
+build() {
+          true
+}
+
+install() {
+          sudo install -D conky/.conkyrc -t /usr/src/pacstall/systemcoding-conky
```
Comment (Henryws) [inline packages/systemcoding-conky/systemcoding-conky.pacscript:20]:
```suggestion
          sudo install -D conky/.conkyrc -t "${STOWDIR}/${name}"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Also run pre-commit hooks.

File: packages/systemcoding-conky/systemcoding-conky.pacscript (hunk 1 above)
Comment (systemcoding) [reply packages/systemcoding-conky/systemcoding-conky.pacscript]: Yea changed the path to ${STOWDIR}/${name}

**Comment by Henryws** [comment]
Fix tabbing.

**Review by Henryws — CHANGES_REQUESTED** [review]
Add `hash`

**Comment by systemcoding** [comment]
hash is not matching please wait while i resolve this issue

File: packages/systemcoding-conky/systemcoding-conky.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name="systemcoding-conky"
+pkgname="conky"
+version="1.0"
+url="https://github.com/systemcoding/systemcoding-conky/releases/download/release/systemcoding-conky.zip"
+maintainer="System Coding <revanthnagmunagala@gmail.com>"
+build_depends="conky lm-sensors"
+gives="conky"
+description="System Coding's Version Of Conky"
+hash="c87be1530a56cfc435ff82cf06a4c6b5914e95bd51944726ae183f4baac08cca"
```
Comment (Henryws) [inline packages/systemcoding-conky/systemcoding-conky.pacscript:9]:
```suggestion
hash="345774e7af5ce7783a1168746243e252922cac7fe0e0b8039c272792492a365c"
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Here's the right hash.

**Review by Henryws — APPROVED** [review]

**Pull request merged**